docs/pages/server-access/guides/auditd.mdx

```diff
@@ -0,0 +1,125 @@
+---
+title: Configure SSH with the Linux Auditing System
+description: How to configure Teleport SSH with auditd (Linux Auditing System).
+h1: Linux Auditing System (auditd)
+---
+
+You can configure Teleport's SSH Service to integrate with the Linux Auditing System (auditd).
+
+## Prerequisites
+
+(!docs/pages/includes/edition-prereqs-tabs.mdx!)
+
+- A running Teleport Node. See the [Server Access Getting Started Guide](../getting-started.mdx) for  how to add a Node to your Teleport cluster.
+- Linux kernel 2.6.6+ compiled with `CONFIG_AUDIT`. Most Linux distributions have this option enabled by default.
+- `auditctl` to check auditd status (optional).
+
+(!docs/pages/includes/tctl.mdx!)
+
+## Step 1/3. Check system configuration
```

Comment:
```suggestion
## Step 1/4. Check system configuration
```
I'm suggesting turning the "How to trace SSH sessions" H2 section into a numbered step to be consistent with the format of our other how-to guides.